dataInterface/items.js

```diff
@@ -9,20 +9,27 @@ module.exports.getAll = () => {
 }
 
 module.exports.getById = (itemId) => {
-  // TODO: complete writing this function
+  return module.exports.items.find(item => item.id == itemId)
 }
 
 module.exports.deleteById = async (itemId) => {
-    // TODO: complete writing this function
+  const index = module.exports.items.findIndex((item) => item.id == itemId)
+  let test = module.exports.items.splice(index, 1)
+  return module.exports.items
+
 }
 
 module.exports.updateById = async (itemId, newObj) => {
-    // TODO: complete writing this function
+  let item = module.exports.items.find(item => item.id == itemId)
+  if(item){
+    item.field = newObj.field
+    return module.exports.items
+  }
```

Comment:
It would be good to have an explicit return value in case the `item` is not found, as well. This logic should be handled here, not in the route handler. If it cannot be found case, the implicit return value of the function will be `undefined`, but it would be preferable to explicitly return `null` or similar, for readability and maintainability.